addons/web_tour/static/src/tour_service/tour_step_automatic.js

```diff
@@ -188,15 +188,22 @@ export class TourStepAutomatic extends TourStep {
      * @returns {HTMLElement}
      */
     findTrigger() {
-        let nodes;
+        let triggerEl;
         try {
-            nodes = hoot.queryAll(this.trigger);
+            //TODO: this condition has to be removed after hoot bug has been fixed
```

Comment:
@Arcasias 
https://media.discordapp.net/attachments/1096043239250853958/1269937553394368583/image.png?ex=66b72703&is=66b5d583&hm=d219b414ecd2ae191d56f8037b09ba4a46b0ed991fdc33ddf97d55598d3eff0b&=&format=webp&quality=lossless&width=607&height=468